modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/GetFormReportDataMVCResourceCommand.java

```diff
@@ -113,10 +133,19 @@ protected void doServeResource(
 	private static final Log _log = LogFactoryUtil.getLog(
 		GetFormReportDataMVCResourceCommand.class);
 
+	@Reference(
+		target = "(model.class.name=com.liferay.dynamic.data.mapping.model.DDMFormInstance)"
+	)
+	private ModelResourcePermission<DDMFormInstance>
+		_ddmFormInstanceModelResourcePermission;
+
 	@Reference
 	private DDMFormInstanceReportLocalService
 		_ddmFormInstanceReportLocalService;
 
+	@Reference
+	private DDMFormInstanceService _ddmFormInstanceService;
+
```

Comment:
Since we will remove that line, I believe this will be removed too.

Reply:
Done